dictionaries/bg_BG/src/bg_BG.dic

```diff
@@ -2123,6 +2123,7 @@
 Илчева
 Илчо
 Империал
+имейл
```

Comment:
@KieranRobson,

`.dic` and `.aff` files get updated from external sources. We will need to add this word to another file or it will get lost.

Reply:
I'll leave it in for now, but it will disappear when those files get updated.